src/services/cluster/listen-command.ts

```diff
@@ -0,0 +1,13 @@
+import cluster from "cluster";
```

Comment:
Am I correctly understand that we use cluster to send our message across all processes which were spawned by BullQueues?